### PR TITLE
Companion for #6912

### PR DESCRIPTION
**Pull request by weichweich**
Companion for https://github.com/paritytech/substrate/pull/6912

### CODE REVIEW

**Review by athei — APPROVED** [review]
LGTM

**Review by thiolliere — APPROVED** [review]

File: runtime/kusama/src/lib.rs [hunk 1]
```diff
@@ -230,7 +231,7 @@ parameter_types! {
 
 impl pallet_transaction_payment::Trait for Runtime {
 	type Currency = Balances;
```
Comment (thiolliere) [inline runtime/kusama/src/lib.rs:233]: this associated type should be now removed

File: runtime/polkadot/src/lib.rs [hunk 2]
```diff
@@ -243,7 +244,7 @@ parameter_types! {
 
 impl pallet_transaction_payment::Trait for Runtime {
 	type Currency = Balances;
```
Comment (thiolliere) [inline runtime/polkadot/src/lib.rs:246]: this associated type should be now removed

**Comment by thiolliere** [comment]
ah can you upgrade substrate ? I merged on substrate

**Comment by thiolliere** [comment]
just do `cargo update -p sp-io`

**Comment by thiolliere** [comment]
bot merge

**Pull request merged**